### PR TITLE
Swiss statistics days 2024

### PR DESCRIPTION
**Pull request by bethcg**

### CODE REVIEW

File: blog/2024-09-11-swiss-statistics-days/index.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+title: |
+  Swiss Statistics Meeting - Renku: a platform for transparent and reproducible data-centric work
+authors: [elisabet, rokroskar, laura, jimena]
+tags:
+  - Swiss Statistics Meeting
+  - Poster
+image: ./image_1.png
+description: |
+  Renku was presented in the Swiss Statistics Meeting, which took place on Sept. 4th
+  and 5th 2024 in Aarau (Switzerland). This meeting gathered specialists related to
```
Comment (lokijuhy) [inline blog/2024-09-11-swiss-statistics-days/index.mdx:12]:
```suggestion
  and 5th 2024 in Aarau (Switzerland) and brought together specialists related to 
```

File: blog/2024-09-11-swiss-statistics-days/index.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  Swiss Statistics Meeting - Renku: a platform for transparent and reproducible data-centric work
+authors: [elisabet, rokroskar, laura, jimena]
+tags:
+  - Swiss Statistics Meeting
+  - Poster
+image: ./image_1.png
+description: |
+  Renku was presented in the Swiss Statistics Meeting, which took place on Sept. 4th
+  and 5th 2024 in Aarau (Switzerland). This meeting gathered specialists related to 
+  statiscis domain across private and public sectors. The SDSC contribution “Renku: a platform
```
Comment (lokijuhy) [inline blog/2024-09-11-swiss-statistics-days/index.mdx:13]:
```suggestion
  statistics domain across private and public sectors. The SDSC contribution “Renku: a platform 
```

File: blog/2024-09-11-swiss-statistics-days/index.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  where data scientists, statisticians and other practitioners can work in their prefererred
+  compute environment (Python, R, Matlab, etc. ) and share their work.
+---
+
+![Renku in Swiss Statistics Meeting 2024](./image_1.png)
+
+<!-- truncate -->
+
+The [Swiss Statistics Meeting](https://stat.ch/en/swiss_statistics_meeting) took place
+on Sept. 4th and 5th 2024 in Aarau (Switzerland). It gathered specialists related to
```
Comment (lokijuhy) [inline blog/2024-09-11-swiss-statistics-days/index.mdx:26]:
```suggestion
on Sept. 4th and 5th 2024 in Aarau (Switzerland) and brought together specialists related to
```

File: blog/2024-09-11-swiss-statistics-days/index.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+the statistics domain across private and public sectors. Multiple presentations contributed
+to discuss developments in research and methodological questions related to data across
+the data lifecycle, including topics such as data sources,
+data collection and management, data analysis, open data, and data security among others.
+
+Our contribution [“Renku: a platform for transparent and reproducible data-centric work”](https://drive.google.com/file/d/1qYsrPm4rm3Osjmjvmoz0RIgWyxWN335F/view?usp=share_link)
+focused on those Renku features in Renku 2.0 that make it a unique platform to support
+transparent data-centric work. Renku connects to public and private data sources, and to
+code repositories. Thus, Renku 2.0 can launch different types of sessions on top of them,
```
Comment (lokijuhy) [inline blog/2024-09-11-swiss-statistics-days/index.mdx:35]:
```suggestion
code repositories. These data and code come to life in Renku sessions,
```

File: blog/2024-09-11-swiss-statistics-days/index.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+where data scientists, statisticians and other practitioners can work in their prefererred
+compute environment (Python, RStudio, Matlab, VSCode or others ) with the compute infrastructure
+that they select. Having these three components under the same umbrella makes it easier to
```
Comment (lokijuhy) [inline blog/2024-09-11-swiss-statistics-days/index.mdx:38]:
```suggestion
that they select. Having data, code, and compute under the same umbrella makes it easier to
```

File: blog/2024-09-11-swiss-statistics-days/index.mdx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  for transparent and reproducible data-centric work” focused on those features in 
+  Renku 2.0 that make it a unique platform to support transparent data-centric work, 
+  since it connects data sources and code repositories, and enables launching sessions, 
+  where data scientists, statisticians and other practitioners can work in their prefererred
+  compute environment (Python, R, Matlab, etc.) and share their work.
+---
+
+![Renku in Swiss Statistics Meeting 2024](./image_1.png)
+
+<!-- truncate -->
```
Comment (leafty) [inline blog/2024-09-11-swiss-statistics-days/index.mdx:23]: Honestly, you should truncate after the first paragraph. This does not look engaging at all:

Screenshot 2024-09-23 at 09 18 44

File: blog/2024-09-11-swiss-statistics-days/index.mdx [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+slug: sst-2024
+title: |
+  Swiss Statistics Meeting - Renku: a platform for transparent and reproducible data-centric work
+authors: [elisabet, rokroskar, laura, jimena]
+tags:
+  - Swiss Statistics Meeting
+  - Poster
+image: ./image_1.png
```
Comment (leafty) [inline blog/2024-09-11-swiss-statistics-days/index.mdx:9]: Image for SEO/link previews should be landscape.

**Review by leafty — DISMISSED** [review]

**Review by lokijuhy — CHANGES_REQUESTED** [review]
There's something off with the image- it renders unusually small.

**Review by lokijuhy — APPROVED** [review]

**Pull request merged**